### PR TITLE
Mehrzad / add popover

### PR DESCRIPTION
**Pull request by mehrzad-fs**

### CODE REVIEW

File: src/botPage/view/deriv/layout/Footer/components/network-status.jsx [hunk 1]
```diff
@@ -35,13 +38,15 @@ const NetworkStatus = ({ api }) => {
...
     return (
         <div id="network-status" className="network-status__wrapper">
-            <div
-                className={classNames("network-status__circle", {
-                    "network-status__circle--offline": status === "offline",
-                    "network-status__circle--online": status === "online",
-                    "network-status__circle--blinker": status === "blinker",
-                })}
-            />
+            <Popover content={<>{translate('Network status:')}{status}</>}>
```
Comment (nikolai-deriv) [inline src/botPage/view/deriv/layout/Footer/components/network-status.jsx:41]:
```suggestion
            <Popover content={<>{translate('Network status: ')}{status}</>}>
```

File: src/botPage/view/deriv/components/popover.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Popover as TinyPopover } from 'react-tiny-popover';
+
+const Popover = ({children,content})=>{
```
Comment (nikolai-deriv) [inline src/botPage/view/deriv/components/popover.jsx:4]:
```suggestion
const Popover = ({ children, content })=>{
```

File: static/css/deriv/popover.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.popover {
+    &__container {
+        position: relative;
+        font-size: 11px;
```
Comment (nikolai-deriv) [inline static/css/deriv/popover.scss:4]:
```suggestion
        font-size: 1.6em;
```

File: static/css/deriv/popover.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        margin: 10px;
+        height: 25px;
+        border-radius: 2px;
```
Comment (nikolai-deriv) [inline static/css/deriv/popover.scss:7]:
```suggestion
        border-radius: 4px;
```

File: static/css/deriv/popover.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        & > span{
+            background: var(--general-hover);
+            padding: 5px 10px;
```
Comment (nikolai-deriv) [inline static/css/deriv/popover.scss:10]:
```suggestion
            padding: 10px 20px;
```

File: static/css/deriv/popover.scss (hunk 3 above)
Comment (mehrzad-fs) [reply static/css/deriv/popover.scss:4]: I'd checked `1.6em` but it's so huge

File: static/css/deriv/popover.scss (hunk 5 above)
Comment (mehrzad-fs) [reply static/css/deriv/popover.scss:10]: this update caused some issues on the popver design

**Pull request merged**